src/main/features/ccj/routes/pay-by-set-date.ts

```diff
@@ -9,9 +9,13 @@ import { DraftService } from 'services/draftService'
 import { ErrorHandling } from 'common/errorHandling'
 import { DraftCCJ } from 'ccj/draft/draftCCJ'
 import { Draft } from '@hmcts/draft-store-client'
+import { MomentFactory } from 'common/momentFactory'
 
 function renderView (form: Form<PayBySetDate>, res: express.Response): void {
-  res.render(Paths.payBySetDatePage.associatedView, { form: form })
+  const futureDate: string = 'For example, ' + MomentFactory.currentDate().add(30, 'days').format('DD MM YYYY')
```

Comment:
the for example should be added in the view, otherwise it can't be translated properly because there's a parameter in the string that is dynamic

Reply:
done but isn't the entire string just passed to the translation function in the macro? so it would get translated?

Reply:
can't translate dynamic parameter thats why we do:
t('adsad {{ param }}', param: 'hi rob')